x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/condition_entry_input/index.tsx

```diff
@@ -149,12 +174,30 @@ export const ConditionEntryInput = memo<ConditionEntryInputProps>(
             />
           </ConditionEntryCell>
         </EuiFlexItem>
-        <EuiFlexItem>
-          <ConditionEntryCell showLabel={showLabels} label={ENTRY_PROPERTY_TITLES.operator}>
-            <EuiFieldText name="operator" value={OPERATOR_TITLE.included} readOnly />
-          </ConditionEntryCell>
-        </EuiFlexItem>
-        <EuiFlexItem grow={3}>
+        {entry.field === ConditionEntryField.PATH ? (
+          <EuiFlexItem style={{ gridArea: 'operator' }}>
```

Comment:
Suggestion: the only part that is different in this code branching here is the component inside of the `<ConditionEntrycell>`, so maybe instead of duplicating the entire `EuiFlexItem` code, just move this condition to branch only between displayng a `EuiSuperSelect` or a `EuiFieldText`

Reply:
done 74d5193b2d9259ae681242aa24efc2e6650e5638 👍